ui_framework/src/components/context_menu/context_menu_panel.js

```diff
@@ -220,7 +234,13 @@ export class KuiContextMenuPanel extends Component {
         this.props.onHeightChange(node.clientHeight);
       }
     }
-  }
+  };
+
+  contentRef = node => {
+    if (node) {
+      this.content = node;
+    }
+  };
```

Comment:
If we store a reference to the dom element here, we should also clear it when we're called with `null`. Just storing `this.content = node` would do that.